qcodes/tests/common.py

```diff
@@ -11,6 +11,7 @@
 from qcodes.utils.metadata import Metadatable
 from qcodes.configuration import Config, DotDict
 
+from _pytest._code.code import ExceptionChainRepr
```

Comment:
I would like to avoid this private import. But since excinfo.getrepr can return two different types I don't see any way around this that does not involve either comparing the types or ignoring the type checking error. The later seems worse